### DISCUSSION TITLE
How to handle different extension versions?

### DISCUSSION THREAD

**Discussion by duckontheweb** [radiantearth/stac-spec#1145] [STAC Software]
Now that STAC extension versions are no longer tied to the core STAC version or to each other, we may run into situations where a user is working with catalogs that implement some combination of a core spec version and extension versions that are not all supported by the same version of PySTAC.

I'm wondering if anyone has started to think through a way to handle versioning of STAC extension implementations PySTAC that:

1) Does not enforce a single combination of versions on a user
2) Is maintainable in the code

@emmanuelmathot How are you handling versioning of STAC extension implementations in DotNetStac?

**Comment by lossyrob** [comment]
Note that the discussion has moved to https://github.com/stac-utils/pystac/issues/448

**Reply by m-mohr** [reply]
Although I think it is not necessarily an issue that only applies to PySTAC. All tooling is basically having this issue and a discussion in the whole community might be required. Could be something for the STAC meeting on Monday.
